### ISSUE TITLE
Rotation Preference attribute is set to "any"

### ISSUE THREAD

**Issue by dmitrymal** [manifoldjs/ManifoldJS#192]
I used http://manifoldjs.com/generator and struggled to get the windows project running using the 
`manifoldjs run windows` command. The powershell would just shutdown without showing any errors.
When I gen'ed the code locally everything worked fine. I tracked the difference maker being the Rotation->Preference attribute in the `appxmanifest.xml` file. It was set to **any** by the online generation tool:
```xml
<uap:InitialRotationPreference>
	<uap:Rotation Preference="any" />
</uap:InitialRotationPreference>
```
Apparently it may not be a valid value. Replacing it with **portrait** solved the problem.

I am not sure what causes that property to be set to "any" when using the online generator. So I decided to post this here to see if people can fix it.

**Comment by boyofgreen** [comment]
Yes this seems to be a issue, I'll make sure this gets addressed.

**Comment by boyofgreen** [comment]
@f2bo if we get a chance to slip this in, let's remove any

**Comment by f2bo** [comment]
Fixed in v0.5.0 (see https://github.com/manifoldjs/ManifoldJS/pull/206)
